Sources/TokamakCore/Animation/Animatable.swift

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 7/11/21.
+//
+
+import Foundation
+
+public protocol Animatable {
+  associatedtype AnimatableData: VectorArithmetic
+  var animatableData: Self.AnimatableData { get set }
+}
+
+public protocol _PrimitiveAnimatable {}
+
+public extension Animatable where Self: VectorArithmetic {
+  var animatableData: Self {
+    get { self }
+    set {}
```

Comment:
Is there some nice way to satisfy the linter here? Or is explicitly silencing `unused_setter_value` the only way here?

Reply:
Doesn't look like I can get around it without using `newValue`:
```swift
set { _ = newValue }
```
I think I'll just disable it manually.